app/models/collection_option.rb

```diff
@@ -2,7 +2,7 @@
 class CollectionOption < ApplicationRecord
   EMAIL_MESSAGES = 0
 
-  belongs_to :collection
+  belongs_to :group, class_name: "Group", foreign_key: "collection_id"
```

Comment:
Should this class be renamed GroupOption, otherwise I would still expect it to have a collection attribute